mava/systems/ff_ippo_rware.py

```diff
@@ -621,11 +618,8 @@ def hydra_entry_point(cfg: DictConfig) -> None:
     # Convert config to python dict.
     cfg: Dict = OmegaConf.to_container(cfg, resolve=True)
```

Comment:
I think we don't need this anymore

Reply:
I agree that we should definitely move from a dict to the default `hydra` `.` syntax but this will take quite a bit of admin since all of Mava expects dicts at the moment. Can we make a maintenance issue for updating to this later on?

Reply:
I made [this issue](https://github.com/instadeepai/Mava/issues/949) so that we can address this in the future.

Reply:
Sounds great!